### PR TITLE
A few more fixes for bug 756533.

### PR DESCRIPTION
**Pull request by jezdez**

### CODE REVIEW

**Comment by jezdez** [comment]
I'm a bit unsure if that will mean that prod and stage will use the debug logger. The whole settings setup is so messed up I don't know what to do

File: settings.py [hunk 1]
```diff
@@ -208,8 +210,22 @@
     if delivered_lang in PROD_LANGUAGES:
         LANGUAGE_URL_MAP[requested_lang.lower()] = delivered_lang
 
-# Override Django's built-in with our native names
+
+def get_locales():
+    locales = {}
+    file = os.path.join(ROOT, 'kuma', 'languages.json')
+    json_locales = json.loads(open(file, 'r').read())
```
Comment (robhudson) [inline settings.py]: Simpler might be to just use `json.load`?

**Comment by jezdez** [comment]
The circular logging filter issue was fixed in Django 1.5 in 2012 (https://github.com/django/django/commit/fc69fff9ab5bba8a6cff3eaf51f02a3204b1c015)

File: settings.py (hunk 1 above)
Comment (jezdez) [inline settings.py]: Done.

File: settings.py [hunk 2]
```diff
@@ -208,8 +210,22 @@
     if delivered_lang in PROD_LANGUAGES:
         LANGUAGE_URL_MAP[requested_lang.lower()] = delivered_lang
 
-# Override Django's built-in with our native names
+
+def get_locales():
+    locales = {}
+    file = os.path.join(ROOT, 'kuma', 'languages.json')
```
Comment (robhudson) [inline settings.py]: On my filesystem this is at `kuma/core/languages.json`. So I would think you need to add the extra `'core'` in there.

File: settings.py [hunk 3]
```diff
@@ -208,8 +210,22 @@
...
+    json_locales = json.load(open(file, 'r'))
+    for locale, meta in json_locales.items():
+        locales[locale] = _Language(meta['english'],
+                                    meta['native'],
+                                    meta['iso639_1'])
```
Comment (robhudson) [inline settings.py]: Generally curious why this needs to be a namedtuple and not just the dict imported from the json file?

File: settings.py (hunk 2 above)
Comment (jezdez) [inline settings.py]: The commit moves the file as well

File: settings.py (hunk 3 above)
Comment (jezdez) [inline settings.py]: I have no idea

File: settings.py (hunk 2 above)
Comment (robhudson) [inline settings.py]: Ah, I see that now. :+1:

**Comment by robhudson** [comment]
r+

**Pull request merged**